pkg/prowgen/testdata/zz_fixture_TestGenerateJobBase_config_variant.yaml

```diff
@@ -3,7 +3,6 @@ decorate: true
 decoration_config:
   skip_cloning: true
 labels:
-  ci-operator.openshift.io/prowgen-controlled: newly-generated
```

Comment:
Why are these removed and not changed to the new label?

Reply:
Generating the job base no longer needs to add the old label or the new `ci.openshift.io/generator` label as that is done in the `jc.WriteToDir` function.